clouseau/patchanalysis.py

```diff
@@ -245,6 +257,21 @@ def attachmenthandler(attachments, bugid, data):
         'r-ed_patches': sum((a['is_patch'] == 1 or a['content_type'] == 'text/x-review-board-request') and sum(flag['name'] == 'review' and flag['status'] == '-' for flag in a['flags']) > 0 for a in bug['attachments']),
     }
 
+    # Store bug creator & assignee
+    assignee = bug.get('assigned_to_detail')
+    creator = bug.get('creator_detail')
+
+    # Search uplift requester & raw request from comment
```

Comment:
If you want to make this `bug_analysis` patch also return the uplift info (which is fine for me), you can move this part to `uplift_info` and then here do something like:
`info.update(uplift_info(bug, channel))`